jetty-core/jetty-server/src/main/config/modules/static-deploy.mod

```diff
@@ -1,7 +1,5 @@
 [description]
-# tag::description[]
-Scans and deploys `static` content from `$JETTY_BASE/webapps/` directory.
-# end::description[]
+Scans and deploys Jetty `static` environment web applications.
```

Comment:
static doesn't really deploy applications, at least not in our sense of the word
```suggestion
Scans and deploys directories that are to be served statically by Jetty.
```

Reply:
@gregw it actually does deploy a web application, in our sense of a `ContextHandler` being made available, with a `contextPath`, etc.
Users won't be able to figure out whether it is static files or generated by the servlet, so they will think they hit a web application deployed on the server.